README.md

```diff
@@ -27,7 +27,7 @@ We will copy your existing files in the below location with a `.bak` extension a
 
 - In the below instructions `demo` is the environement name. You can change it as per your need. For example - `dev`, `stage` etc.
 - Clone the repo
-`git cline git@github.com:nimbushubin/sunbird.git`
+`git clone git@github.com:nimbushubin/sunbird.git`
```

Comment:
Update this instruction to use sunbird-ed-installer as repository. Otherwise git clone will not work